### PR TITLE
Add domain name in url in web tests

### PR DESCRIPTION
**Pull request by AlekseyDidilov**
Web test was updated, url with 'local' domain names was replaced
on 'vm' domain name, for ability to run on remote vm.
Device's IP was replaces on testbed file.
80 port was added in docker-compose.yaml for ability to use
application with http port.

#67

### CODE REVIEW

**Comment by alexbeatnik** [comment]
Good job!

File: testbed.yaml [hunk 1]
```diff
@@ -4,9 +4,9 @@ testbed:
       selenium-grid: http://localhost:9515
     servers:
       server_alias:
-        address: 192.168.195.143
-        username: adminaccount
-        password: Install_new!
+        address: 10.4.137.160
```
Comment (extsoft) [inline testbed.yaml]: @AlekseyDidilov please revert changes to this file.

Comment (AlekseyDidilov) [reply testbed.yaml]: @extsoft Done.

File: oct/tests/deployment/deploy_app.py [hunk 2]
```diff
@@ -14,7 +14,7 @@ def copy_file_to_server(device: Device) -> None:
     futils = FileUtils(testbed=device.testbed)
     futils.copyfile(
         source=f"{os.path.join(os.getcwd(), 'docker-compose.yaml')}",
-        destination=f"scp://{device.connections.main.ip}/home/adminaccount/oct",
+        destination=f"scp://{device.connections.main.ip}/home/alexbeatnik/oct",
```
Comment (GoshkaEgoshka) [inline oct/tests/deployment/deploy_app.py]: Please, change computer name, alexbeatnik->adminaccount

**Comment by GoshkaEgoshka** [comment]
@AlekseyDidilov in some tests you use http, but docker-compose work just with 443 port. We need to use https

File: oct/tests/deployment/deploy_app.py (hunk 2 above)
Comment (AlekseyDidilov) [reply oct/tests/deployment/deploy_app.py]: Thank, fixed.

**Comment by AlekseyDidilov** [comment]
> @AlekseyDidilov in some tests you use http, but docker-compose work just with 443 port. We need to use https

Thank, fixed.

**Pull request merged**